src/tools/ScenarioMeasurement/Directory.Packages.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="System.CommandLine.DragonFruit" Version="0.4.0-alpha.22272.1" />
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageVersion Include="Microsoft.DotNet.PlatformAbstractions" Version="2.1.0" />
```

Comment:
I don't see either of these packages being used in the csproj files below. Why do we have them specified?

Reply:
Probably copypasta error. Fixing...